### ISSUE TITLE
Create option for save log_file.

### ISSUE THREAD

**Issue by develnk** [AuthorizeNet/sdk-php#125]
Hello.
In many place of code have
```
file_put_contents($this->_log_file, "\nSending 'XML' Request type", FILE_APPEND);
```
And no possibility make so that not save data in log_file.
While we offer use _define("AUTHORIZENET_LOG_FILE", "autorize");_ but it only setting path and filename for save data.
Would be create method similar on:
```
private function saveLogFile($message, $add_message = "") {
    if ($this->_log_file) {
      file_put_contents($this->logFile, sprintf($message, $this->now(), $add_message), FILE_APPEND);
    }
  }
```
And save data in to file: $this->saveLogFile("\n%s: Request  Serialization End");

**Comment by ashtru** [comment]
If you do not want to save any data in log file: remove` define("AUTHORIZENET_LOG_FILE", "autorize");`
If you want to save specific lines into specific file:
```
      $logger = \net\authorize\util\LogFactory::getLog("charge-credit-card");
      $logger->setLogFile("custom-log-file");
      $logger->info("\n%s: Request Serialization Ends");
```
Does that answer your question?

**Comment by kikmak42** [comment]
This is the old method of logging we have been using
```
file_put_contents($this->_log_file, "\nSending 'XML' Request type", FILE_APPEND);
```
but now we are using a new logger class which make it a one step for logging.
  
as @ashtru has mentioned for setting a log file you need
```
 $logger = \net\authorize\util\LogFactory::getLog("charge-credit-card");
 $logger->setLogFile("custom-log-file");
```
and for logging anything in info level use : 
```
$logger->info("\n%s: Log : this is logged");
```

**Comment by ashtru** [comment]
Fixed by #168. Closing this issue, please reopen in case of doubts.
